packages/openapi3/package.json

```diff
@@ -44,15 +47,21 @@
     "test:ci": "vitest run --coverage --reporter=junit --reporter=default",
     "lint": "eslint . --max-warnings=0",
     "lint:fix": "eslint . --fix",
-    "regen-docs": "tspd doc .  --enable-experimental  --output-dir ../../docs/emitters/openapi3/reference"
+    "regen-docs": "tspd doc .  --enable-experimental  --output-dir ../../docs/emitters/openapi3/reference",
+    "regen-specs": "cross-env RECORD=true vitest run",
+    "gen-version": "node scripts/generate-version.js"
   },
   "files": [
     "lib/*.tsp",
     "dist/**",
     "!dist/test/**"
   ],
   "dependencies": {
-    "yaml": "~2.4.5"
+    "@readme/openapi-parser": "~2.6.0",
+    "@typespec/prettier-plugin-typespec": "workspace:~",
+    "prettier": "~3.2.5",
```

Comment:
I see you actually use `formatTypeSpec` from compiler do you need to take a dependency on  prettier and the prettier plugin?

Reply:
No - I thought I had removed this but think it got added back in during my stashing/merge conflict resolutioning with `main`